redwood-layout-api/src/commonMain/kotlin/app/cash/redwood/layout/api/properties.kt

```diff
@@ -15,17 +15,16 @@
  */
 @file:Suppress("MemberVisibilityCanBePrivate")
 
-package app.cash.redwood.layout
+package app.cash.redwood.layout.api
 
 import androidx.compose.runtime.Immutable
 import androidx.compose.runtime.Stable
-import kotlin.jvm.JvmInline
 import kotlinx.serialization.Serializable
 
 /** Equivalent to `justify-content`. */
-@JvmInline
+@Immutable
 @Serializable
-public value class MainAxisAlignment private constructor(public val ordinal: Int) {
+public /*value*/ class MainAxisAlignment private constructor(private val ordinal: Int) {
```

Comment:
Non-JVM Compose can't handle these as value classes. Upstream issue forthcoming.

Reply:
Can we work-around it by making the constructor `internal` or does it fail for all value classes?

Reply:
`internal` works!